docs/sankey.md

```diff
@@ -0,0 +1,109 @@
+# Sankey
+
+Note: This component is in alpha.
+
+### Usage
+
+```jsx
+import {Sankey} from 'react-vis';
+
+const nodes = [{name: 'a'}, {name: 'b'}, {name: 'c'}];
+const links = [
+  {source: 0, target: 1, value: 10},
+  {source: 0, target: 2, value: 20},
+  {source: 1, target: 2, value: 20}
+];
+
+<Sankey
+  nodes={nodes}
+  links={links}
+  width={200}
+  height={200}
+/>
+```
+
+### Api
+
+##### width (required)
+##### height (required)
+##### nodes (required)
+
+An array of objects matching the following shape:
+
+```
+{
+  color?: String,
+  opacity?: Number,
+  key?: String
+}
+```
+
+All these fields are optional.
+
+##### links (required)
+
+An array of objects matching the following shape:
+
+```
+{
+  source: Number,
```

Comment:
Also, i feel like using indices might be a little bit error prone? In the future I think it would be good if we could attach a foreign key/local key inter-relationship between the nodes and links